backend/resolvers/mutations/entry.js

```diff
@@ -42,22 +43,49 @@ const createEntry = (entry, entryType, entryId, t) => {
     })
 }
 
+// Rejects the promise if the supplied args indicate the student is doing a
+// single (non-group) submission, but they have met their limit.
+const ensureCanMakeMoreSingleEntries = (
```

Comment:
Nit: I'm being picky here, but I don't think 'ensure' needs to be part of the function name